tests/cpp/ir/ndarray_kernel.cpp

```diff
@@ -0,0 +1,48 @@
+#include "tests/cpp/ir/ndarray_kernel.h"
+
+namespace taichi {
+namespace lang {
+
+std::unique_ptr<Kernel> setup_kernel1(Program *prog) {
```

Comment:
Will test fixture make this cleaner? https://google.github.io/googletest/primer.html#same-data-multiple-tests